docs/docs/index.md

```diff
@@ -81,8 +81,6 @@ $ pdm --pep582 >> ~/.bash_profile
 
 Once again, Don't forget to restart the terminal session to take effect.
 
-**This setup may become the default in the future.**
-
```

Comment:
There's nothing that might become the default there anymore, is it?  (it came initially with 1e0b20f (PYTHONPEP582=1))
Or is it referring to `__pypackages__` being picked up by default, and the setup via `pdm --pep582` becoming obsolete then?

Reply:
Yes, please remove this line

Reply:
Done.